tests/FacebookTest.php

```diff
@@ -36,9 +36,12 @@
 
 class FacebookTest extends \PHPUnit_Framework_TestCase
 {
+    const DEFAULT_GRAPH_VERSION = 'v2.8';
```

Comment:
I'd remove this constant and hardcode it's value everywhere it's used. In tests, we do not need to maintain a consistent value across all tests.
BTW, that why I'd change the value to `vX.X` or `v4.2` (haha), remember to change the value each time the Graph API gets updated is a pain.